packages/@react-types/shared/src/inputs.d.ts

```diff
@@ -12,7 +12,7 @@
 
 import {ReactNode} from 'react';
 
-export type ValidationState = 'valid' | 'invalid';
+export type ValidationState = 'valid' | 'invalid' | null;
```

Comment:
Didn't notice this before. This is used in a ton of places. Are we sure about changing the definition here, or should we use `ValidationState | null` where this is referenced?